### PR TITLE
Add tag count and tag modal components

### PR DESCRIPTION
**Pull request by ryandillinfelton**
4 tag count components on their own
tagCountComponents
Simple tag modal with sample data
simpleModalOpen

*NEW* tag look (the blue one is just me testing that the className prop works)
tagCountComponentsNew
**NEWEST** tag look
tagCountFlexCenter

### CODE REVIEW

File: packages/components/src/Components/TagModal/TagModal.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        return(
+            <Modal
+                {...props}
+                className={classNames('ins-c-tag-modal', className)}
+                isOpen={isOpen}
+                title={`Tags for ${systemName}`}
+                onClose={toggleModal}
+                isFooterLeftAligned
+            >
+                {this.props.children}
```
Comment (ryelo) [inline packages/components/src/Components/TagModal/TagModal.js]: Do you want to throw this up in the `const` above to keep it consistent with the other `this.props` stuff?

**Comment by ryelo** [comment]
@karelhala this LGTM, lemme know if you see anything else so we can get this thing in! :shipit:

**Comment by ryelo** [comment]
@ryandillinfelton Nice job with this PR! 💯

**Comment by karelhala** [comment]
Released in new version [@redhat-cloud-services/frontend-components/v/0.0.19](https://www.npmjs.com/package/@redhat-cloud-services/frontend-components/v/0.0.19)

**Pull request merged**